### PR TITLE
ass(1,2,3)@vaishnavi

### PR DESCRIPTION
**Pull request by vaishnavi2002**
Fixes #

## Proposed Changes

-
-
-

<a href="https://gitpod.io/#https://github.com/aravindvnair99/ACM-Amrita-2019-2020-Assignments/pull/45"></a>

### CODE REVIEW

File: Vaishnavi/ass3.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            <label>Gender</label>
+            <input type="radio" name="gender" value="Male" />Male
+            <input type="radio" name="gender" value="Female"/>Female
+            <input type="radio" name="gender" value="others"/>others<br/>
+            <label>Email</label>
+            <input type="email" name="email"/>
+<label>Password</label>
+            <input type="password" name="password"/>
+            <label>Birthday</label>
+            <input type="date"/>
```
Comment (VinayakTekade) [inline Vaishnavi/ass3.html]: `required` attribute missing

File: Vaishnavi/ass3.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            <input id="user" name="user last name" type="text" required />
+            <br/>
+            <label>Gender</label>
+            <input type="radio" name="gender" value="Male" />Male
+            <input type="radio" name="gender" value="Female"/>Female
+            <input type="radio" name="gender" value="others"/>others<br/>
+            <label>Email</label>
+            <input type="email" name="email"/>
+<label>Password</label>
+            <input type="password" name="password"/>
```
Comment (VinayakTekade) [inline Vaishnavi/ass3.html]: Given condition is not satisfied, refer conditions.txt

**Review by VinayakTekade — CHANGES_REQUESTED** [review]

File: Vaishnavi/ass3.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Register</h1>
+        <form>
+            <label>First Name</label>
+            <input id="user" name="user first name" type="text" required />
+            <label>Last Name</label>
+            <input id="user" name="user last name" type="text" required />
```
Comment (aravindvnair99) [inline Vaishnavi/ass3.html]: Missing `autocomplete` and also, you can't have two entities with same id.

File: Vaishnavi/ass1.html [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        <hr/>
+        <ul>
+            <li>Smith, J. Y. (1985-11-23). "Merlo J. Pusey Dies at 83". The Washington Post. ISSN 0190-8286. Retrieved 2016-03-24.</li>
+               <li> Upi (1985-11-24). "Merlo J. Pusey Dies; Justice's Biographer Won a Pulitzer Prize". The New York Times. ISSN 0362-4331. Retrieved 2016-03-24.</li>
+               <li> Latter-day Saint / Mormon Pulitzer Prize Winners". www.adherents.com. Retrieved 2016-03-24.</li>
+            </ul>
+        
+        
+</body>
+</html>
```
Comment (aravindvnair99) [inline Vaishnavi/ass1.html:46]: Align the code for all 3 files. There's a refactor option in VS Code itself.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

File: Vaishnavi/ass3.html [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Register</h1>
+        <form autocomplete="on">
+            <label>First Name</label>
+            <input id="user" name="user first name" type="text" required />
+            <label>Last Name</label>
+            <input id="user name" name="user last name" type="text" required />
```
Comment (aravindvnair99) [inline Vaishnavi/ass3.html]: `id` can't have spaces.

File: Vaishnavi/ass3.html [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Register</h1>
+        <form autocomplete="on">
```
Comment (aravindvnair99) [inline Vaishnavi/ass3.html]: Use specific `autocomplete` for each entity.

File: Vaishnavi/ass3.html (hunk 2 above)
Comment (aravindvnair99) [reply Vaishnavi/ass3.html]: @vaishnavi2002 This is still pending.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

File: Vaishnavi/ass3.html (hunk 1 above)
Comment (aravindvnair99) [reply Vaishnavi/ass3.html]: @vaishnavi2002 This is still pending.

File: Vaishnavi/ass1.html (hunk 4 above)
Comment (VinayakTekade) [reply Vaishnavi/ass1.html:46]: Press Shift+Alt+F in VS Code to indent your code.

File: Vaishnavi/ass3.html [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+            <input id="username" name="user last name" type="text" required />
+            <br/>
+            <label>Gender</label>
+            <input type="radio" name="gender" value="Male" />Male
+            <input type="radio" name="gender" value="Female"/>Female
+            <input type="radio" name="gender" value="others"/>others<br/>
+            <label for="email">your Email adress:</label>
+            <input type="text" name="email" placeholder="email200.123@gmail.com"/>
+<label>Password</label>
+            <input type="password" name="password" pattern="^[a-zA-Z][a-zA-Z0-9-_\.]{5,15}$" title="min length is 5 and max is 15"/>
```
Comment (VinayakTekade) [inline Vaishnavi/ass3.html]:
```suggestion
            <input type="password"  placeholder="Password" minlength="5" maxlength="15" name="password" autocomplete="new-password">
```

Comment (VinayakTekade) [reply Vaishnavi/ass3.html]: This is the correct way to apply the min and max length of the password.

File: Vaishnavi/ass3.html [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Register</h1>
+        <form autocomplete="off">
+            <label>First Name</label>
+            <input id="user" name="user first name" type="text" required />
```
Comment (VinayakTekade) [inline Vaishnavi/ass3.html:13]: Missing `autocomplete`

File: Vaishnavi/ass3.html [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+            <label>Last Name</label>
+            <input id="username" name="user last name" type="text" required />
```
Comment (VinayakTekade) [inline Vaishnavi/ass3.html:15]: Missing `autocomplete`

File: Vaishnavi/ass3.html [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+            <br/>
+            <label>Gender</label>
+            <input type="radio" name="gender" value="Male" />Male
+            <input type="radio" name="gender" value="Female"/>Female
+            <input type="radio" name="gender" value="others"/>others<br/>
+            <label for="email">your Email adress:</label>
+            <input type="text" name="email" placeholder="email200.123@gmail.com"/>
```
Comment (VinayakTekade) [inline Vaishnavi/ass3.html:22]: Missing `autocomplete`. Write only Email in the placeholder.

**Review by VinayakTekade — APPROVED** [review]

**Review by aravindvnair99 — COMMENTED** [review]
@vaishnavi2002  Merging and closing the pull request for now. Make the necessary changes in another pull request and send it over from a different branch. Also, make sure to fill up the pull request template and also refactor the code henceforth.

**Pull request merged**